### ISSUE TITLE
Support for different config file by server

### ISSUE THREAD

**Issue by blackfist** [blackfist/windmill#2]
It would be cool if we could have different config files available for servers.

Idea for implementing: Each "class" of server would configure their config_tls_endpoint differently. For example `--config_tls_endpoint=/api/config/web

In code we could have something like this

```
post '/api/config' do
  # return default config
end

post '/api/config/:class' do
  get_config(:class)
end

def get_config(class)
  if class == "web"
    # return the web server config
  end
end
```

**Comment by blackfist** [comment]
closed in 3f0bd73223f1fc2847e655a8be367b1bdd98d92d

**Comment by sroberts** [comment]
AHhh nice work around.
